test/index.js

```diff
@@ -92,6 +96,14 @@ const moduleTests = Promise.all ([
   testModule ('test/harmony/index.js', {silent: true})
 ]);
 
+testCommand ('bin/doctest', {
+  status: 1,
+  stdout: '',
+  stderr: unlines ([
+    'error: No files for doctesting provided'
```

Comment:
Because the errors are now generated by the `doctest` function, giving no files would mean seeing no errors. This error ensures that you'll still see what went wrong when no files are provided.